.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,56 @@
+name: test-windows
+on:
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "**.jpg"
+      - "**.png"
+      - "**.gif"
+      - "**.svg"
+      - "adr/**"
+      - "docs/**"
+
+# Abort prior jobs in the same workflow / PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  validate:
+    runs-on: windows-latest
+    steps:
+      - name: "Dependency: Install Golang"
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+
+      - name: "Checkout Repo"
+        uses: actions/checkout@v3
+
+      - name: Use GNU tar for Windows
+        shell: cmd
+        run: |
+          echo "Adding GNU tar to PATH"
+          echo C:\Program Files\Git\usr\bin>>"%GITHUB_PATH%"
+
+      # https://github.com/actions/cache/blob/main/workarounds.md#improving-cache-restore-performance-on-windows-using-cross-os-caching
+      - name: "Setup caching"
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: "Build CLI"
+        shell: pwsh
+        run: make build-cli-windows-amd
```

Comment:
It would be good to document the dependencies needed for this to run on windows somewhere in the developer guide since `make` and many of the `make` targets will not work OOB if someone tries this for themselves (even on pwsh 7).

Reply:
Yeah I'm a bit torn if we really need to actually support developing the zarf codebase on windows too. We have no demand signal for that right now.